spec/views/preview/preview/index.html.erb_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe 'preview/preview/index', type: :view do
+  let(:endpoint) { 'https://galc.oski.cat' }
+
+  it 'sets the API endpoint' do
+    assign(:api_url, endpoint)
+
+    render
+
+    expect(rendered).to match(/#{endpoint}/)
```

Comment:
I don't know if we want to go this deep, but a more specific test might be

```suggestion
    assert_dom 'div[id=?][data-api-base-url=?]', 'galc-app', endpoint
```

I always need to look at docs for the attribute matchers in `assert_dom` but I _think_ this is correct as-written.

Reply:
Ooh, I just thought, since this API-only we may not have `assert_dom` available at all.  In that case, never mind.

Reply:
it's weird that we have `assert_select` available but not `assert_dom`. anyway, this is updated in f7175ff.